imap/jmap_ical.c

```diff
@@ -1489,6 +1891,23 @@ static json_t *participant_from_ical(icalproperty *prop,
         }
     }
 
+    /* scheduleStatus */
+    param = icalproperty_get_first_parameter(prop, ICAL_SCHEDULESTATUS_PARAMETER);
+    if (param) {
+        json_t *jschedstat = json_array();
+        const char *str = icalparameter_get_schedulestatus(param);
+        strarray_t *vals = strarray_split(str, ",", STRARRAY_TRIM);
+        if (strarray_size(vals)) {
+            int i;
+            for (i = 0; i < strarray_size(vals); i++) {
+                json_array_append_new(jschedstat,
+                        json_string(strarray_nth(vals, i)));
+            }
+        }
+        strarray_free(vals);
+        if (jschedstat) json_object_set_new(p, "scheduleStatus", jschedstat);
```

Comment:
When will `jschedstat` be NULL?

Reply:
Thanks. Fixed in https://github.com/cyrusimap/cyrus-imapd/commit/2fdcbc49841c21f05570e9132b0cfb28411c9325